pyoos/parsers/ioos/one/describe_sensor.py

```diff
@@ -49,7 +56,7 @@ def get_ioos_def(self, ident, elem_type, ont):
 
     def __init__(self, element):
         """ Common things between all describe sensor requests """
-        if isinstance(element, str):
+        if isinstance(element, bytes):
```

Comment:
Yes, the etree testing can not really be counted on because of that.  Unless you grab the `type(element)` and look at what module it comes from?  Maybe that's a workaround?